src/audio/csplayer.c

```diff
@@ -534,7 +534,7 @@ static void __CSPHandleMIDIMsg(ALCSPlayer* seqp, ALEvent* event) {
              * velocity (Should never happen with compact midi sequence,
              * but could happen with real time midi.)
              */
-
+        // fall-through
```

Comment:
Maybe add `#define FALLTHROUGH __attribute__((fallthrough))` to macros.h and use it here (and for the other fallthrough cases)? It's hard to know that the comment is suppressing a compiler warning